src/components/NavBar/NavBarLinks.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link, useLocation } from "react-router-dom";
+import { useEffect } from "react";
+
+type NavBarLinksProps = {
+  links: {
+    name: string;
+    path: string;
+  }[];
+  setIsHamburgerMenuOpen: (isOpen: boolean) => void;
+};
+const NavBarLinks = ({ links, setIsHamburgerMenuOpen }: NavBarLinksProps) => {
+  const location = useLocation();
+
+  useEffect(() => {
+    console.log(location.pathname);
+  }, [location]);
+
+  return (
+    <div className="nav-links">
```

Comment:
This could div could probably be removed if you move the styling to the child element